example/eject.js

```diff
@@ -16,20 +17,20 @@
 
 'use strict';
 
-const mountutils = require('.');
-const drive = process.argv[2];
+const mountUtils = require('..');
 
-if (!drive) {
-  console.error(`Usage: ${process.argv[0]} ${process.argv[1]} <drive>`);
+let argv = process.argv.slice(2);
+let disk = argv.shift();
```

Comment:
Only a minor question, but why has the variable changed from `drive` to `disk`, and yet the usage-prompt still mentions `drive`?